### ISSUE TITLE
 Add support for KNX DPT 20.102

### ISSUE THREAD

**Issue by TinusK** [openhab/openhab1-addons#5173]
OpenHAB version 2.0.0 (Release Build)
binding-knx1 version 1.9.0

Item:
```
String Heating_MF_Living_HVAC
	"HVAC"
	(gMF_Living, gHVAC)
	["HVAC"]
	{ knx="20.102:3/3/81" }

```
OpenHAB Log:
```
[ERROR] [el.item.internal.GenericItemProvider] - Binding configuration of type 'knx' of item 'Heating_MF_Living_HVAC' could not be parsed correctly.
org.eclipse.smarthome.model.item.BindingConfigParseException: DPT 20.102 is not supported by the KNX binding.
```

ETS (data on the knx bus):
```
#	Time	Service	Flags 	Prio	Source Address	Source Name	Destination Address	Destination Name	Rout	Type	DPT	Info
6057	4/25/2017 08:03:21.555	to bus		Low	1.2.253	-	3/3/81	MF - Dining + Living	6	Read		$00
6058	4/25/2017 08:03:21.606	from bus		Low	1.2.41	AKH-0400.02 Heating actuator 4-fold, 2TE,24/230VAC	3/3/81	MF - Dining + Living	6	Response	  20.102 HVAC mode	$02 | Standby
```

If I remove the '20.102:' from the binding
```
String Heating_MF_Living_HVAC
	"HVAC"
	(gMF_Living, gHVAC)
	["HVAC"]
	{ knx="3/3/81" }
```
then I get the following error from OpenHAB:
`10:07:03.367 [INFO ] [g.knx.internal.dpt.KNXCoreTypeMapper] - Translator couldn't parse data for datapoint type '16.001' (KNXIllegalArgumentException).`

The above error message remains the same for Item type String or Number.

KNX Datapoint Type 20.102 can have the following data:
$00 Auto
$01 Comfort
$02 Standby
$03 Economy
$04 Building Protection

Maybe the KNX DPT 20.102 could be added?

Thank you,
Tinus

**Comment by TinusK** [comment]
I solved it by using DPT 5.005 iso 20.102:
```
Number Heating_MF_Living_HVAC
	"HVAC"
	(gMF_Living, gHVAC)
	["HVAC"]
	{ knx="5.005:3/3/81" }
```
This produces a 0, 1, 2, 3 or 4 number.

Tinus

**Comment by TinusK** [comment]
I therefor close this issue.

Tinus
